src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -380,9 +388,87 @@ You can also query for all existing streams. Here's an example:
+    1. Navigate to [this page](https://docs.aws.amazon.com/AmazonS3/latest/userguide/olap-examples.html#olap-examples-3), click the link to the serverless repo.
+    2. Click the **Deploy** button at the bottom of the page.
+    3. [Set up an access point on your S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-access-points.html).
+    4. [Create an Object Lambda access point](https://docs.aws.amazon.com/AmazonS3/latest/userguide/olap-create.html). This access point must have these settings:
+    * The **Supporting Access Point** on this Lambda access point will need to be set to the access point you set up on the S3 bucket.
+    * Under **Transformation Configuration**:
+    * The **GetObject** box must be checked.
+    * The DecompressGZFunction Lambda function (or whichever other function is necessary, if a different compression format is used) must be specified.
+  </Collapser>
+
+  <Collapser id="collapser-2" title="Metadata-setting Lambda function">
+    AWS will automatically decompress objects downloaded from S3, if those objects have the correct metadata set. We have written a function that automatically applies this metadata to every new object downloaded to a set S3 object. Here's how to set it up:
+    1. Navigate [here](https://github.com/newrelic/metadata-setting-lambda-function), clone the repository locally, and follow the provided steps in the README file to generate a ZIP file containing the lambda function.
+    2. Create an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) for the function.
+    * When [creating the role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html#roles-creatingrole-service-console), be sure to set the trusted entity type as "AWS Service", with "Lambda" as your use case.
+    * This role must have a policy with at least these permissions: `s3:PutObject` and `s3:GetObject`.
+    3. Navigate to the [Lambda functions page](https://console.aws.amazon.com/lambda/home#/functions) in AWS
+    4. Click the **Create function** button.
+    5. Select the Java 11 runtime environment.
+    6. Click **Change default execution role**, then select **Use an existing role**. Enter the role you created in step 2 here.
+    7. Scroll down and click the **Create function** button.
+    8. Once the function has been created, click **Upload from**, and select **.zip or .jar file** from the dropdown.
+    9. Click **Upload** from the box that pops up, and select the ZIP file created in step 1.
+    10. Once the upload has finished, click **Save** to exit the pop-up box.
+    11. All that's left to do now is enable this lambda function to trigger on S3 object creation. Click **Add trigger** to start setting that up.
```

Comment:
```suggestion
    11. All that's left to do now is enable this lambda function to trigger S3 object creation. Click **Add trigger** to start setting that up.
```

Reply:
This would unfortunately change the meaning of the sentence, we want the Lambda function to be triggered by S3 object creation, not to trigger it itself. Maybe `to be triggered by S3 object creation` would be more clear?